src/main/resources/static/js/subscriptions.js

```diff
@@ -373,7 +373,10 @@ jQuery(document).ready(function () {
                     "targets": [6],
                     "orderable": true,
                     "title": "NotificationMeta",
-                    "data": "notificationMeta"
+                    "data": "notificationMeta",
+                    "render": function (data, type, row, meta) {
+                        return encodeURI(data);
```

Comment:
Is this enough? If you check https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/encodeURI it will not escape `A-Z a-z 0-9 ; , / ? : @ & = + $ - _ . ! ~ * ' ( ) #` where as https://cheatsheetseries.owasp.org/cheatsheets/Cross_Site_Scripting_Prevention_Cheat_Sheet.html recomends to escape
```
 & --> &amp;
 < --> &lt;
 > --> &gt;
 " --> &quot;
 ' --> &#x27;     
 / --> &#x2F;    
```